src/System.Runtime.Serialization.Formatters/tests/BinaryFormatterTestData.cs

```diff
@@ -637,24 +1351,24 @@ private static IEnumerable<object[]> SerializableObjects()
-            }
+            //if (PlatformDetection.IsNetfx471OrNewer())
```

Comment:
commented out?

Reply:
This is currently failing on machines that already have net471 installed (win10 fall creators update ones) because we are currently compiling core still against net461 which produces shims that don't typeforward correctly to the right package. Therefore I decided to disable it here with this PR and reenable as soon as we compile against net471.

Reply:
Assuming we service this change, presumably it will go out built against net461. Will there be implications for customers who have net471 (or don't)?